src/test/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAOTest.java

```diff
@@ -7,6 +7,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.sql.Timestamp;
+import java.time.Instant;
```

Comment:
This looks unused

Reply:
Surprised Sonar didn't catch that.  Thanks!